libs/NfcCoreLib/lib/LibNfc/phLibNfc_Se.c

```diff
@@ -1329,6 +1383,73 @@ static NFCSTATUS phLibNfc_SetSeModeSeqComplete (void* pContext,NFCSTATUS Status,
     return wStatus;
 }
 
+static NFCSTATUS
+phLibNfc_SePowerAndLinkCtrlSeq(void *pContext,
+                               NFCSTATUS wStatus,
+                               void *pInfo)
+{
+    NFCSTATUS wIntStatus;
```

Comment:
You should initialize 'wIntStatus' to a valid value.

Reply:
I have assigned wIntStatus at declaration to NFCSTATUS_FAILED.